cli/azd/cmd/show.go

```diff
@@ -165,7 +178,82 @@ func (s *showAction) Run(ctx context.Context) (*actions.ActionResult, error) {
 		}
 	}
 
-	return nil, s.formatter.Format(res, s.writer, nil)
+	if s.formatter.Kind() == output.JsonFormat {
+		return nil, s.formatter.Format(res, s.writer, nil)
+	}
+
+	appEnvironments, err := s.envManager.List(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	uxEnvironments := make([]*ux.ShowEnvironment, len(appEnvironments))
+	for index, environment := range appEnvironments {
```

Comment:
nit: Not sure if you like it better, but one way to not introduce an additional variable for `index` here for just purpose of iteration, is to do:

```golang
uxEnvironments := make([]*ux.ShowEnvironment, 0, len(appEnvironments))
for _, environment := range appEnvironments {
  uxEnvironments = append(uxEnvironments, &ux.ShowEnvironment{})
}
```

Which I find slightly neater and more robust since we don't care about matching indices, just copying of collection. Same comment applies for `ux.ShowService` below.

Reply:
most likely it will be the same result after compilation.  
My preference goes against using the `0` as magic number there and the `_` to ignore the output.